src/env.ts

```diff
@@ -0,0 +1,16 @@
+declare var window: any
```

Comment:
The result of `.match()` below can be `null`.  You are bypassing that check because of this line. Please remove this and check the result of `match` before accessing it:

```ts
function getTLD() {
  const result = window.location.hostname.match(/(\w+)$/)
  return result ? result[0] : null
}
```